corehq/messaging/scheduling/forms.py

```diff
@@ -1822,8 +1881,7 @@ def get_after_content_layout_fields(self):
                 _("Advanced Survey Options"),
                 *self.get_advanced_survey_layout_fields(),
                 data_bind=(
-                    "visible: content() === '%s' || content() === '%s'" %
-                    (self.CONTENT_SMS_SURVEY, self.CONTENT_IVR_SURVEY)
+                    f"visible: content() === '{self.CONTENT_SMS_SURVEY}' || content() === '{self.CONTENT_IVR_SURVEY}' || content() === '{self.CONTENT_CONNECT_SURVEY}"
```

Comment:
I think you forgot a `'` at the end there (currently breaking conditional alert form)
```suggestion
                    f"visible: content() === '{self.CONTENT_SMS_SURVEY}' || content() === '{self.CONTENT_IVR_SURVEY}' || content() === '{self.CONTENT_CONNECT_SURVEY}'"
```